src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/DisputeService.java

```diff
@@ -92,8 +100,12 @@ public Dispute update(Long id, Dispute dispute) {
 		}
 		// Add updated ticket counts
 		disputeToUpdate.addDisputeCounts(dispute.getDisputeCounts());
-
-		return disputeRepository.save(disputeToUpdate);
+		
+		Dispute updatedDispute = disputeRepository.saveAndFlush(disputeToUpdate);
+		// We need to refresh the state of the instance from the database in order to return the fully updated object after persistance 
+		entityManager.refresh(updatedDispute);
```

Comment:
Out of curiosity, would disputeRepository.getDisputeById(updatedDispute.getId()) work here? From my understanding, the base object is refreshed (which includes the id and other primitives), just not the child objects, right?  In which case you can get rid of the enityManager and use use the same repository to refresh the object.

Reply:
That makes sense to me as well. However, I tried the way you suggested and tried refreshing the returned object through querying it by id using the same repository (disputeRepository) but it still returns the "createdBy" and "createdTs" fields of the child objects as null unfortunately. It might be an issue with H2 database, I'm not sure. So will keep the entityManager for now. 
![jj-response](https://user-images.githubusercontent.com/98848668/189710873-1b13f704-e513-479d-babc-65cb24c087a7.png)

Reply:
Ah, okay.